include/fi_util.h

```diff
@@ -376,12 +373,18 @@ int ofi_av_get_index(struct util_av *av, const void *addr);
  * Connection Map
  */
 
+// TODO explore replacing this with a simple connection hash map that is common
+// for both AV and RX only connections.
 #define UTIL_CMAP_IDX_BITS 48
 
 enum util_cmap_state {
 	CMAP_UNSPEC,
-	CMAP_CONNECTING,
-	CMAP_CONNECTED
+	CMAP_CONN_INIT,
+	CMAP_CONNECT,
```

Comment:
Yes this state would correspond to CONNREQ_SENT. I think the CONN_INIT state could be done away with. It was initially added on the first iteration of this change when it was needed back then.